lib/content-services/document-list/services/document-actions.service.spec.ts

```diff
@@ -204,19 +204,6 @@ describe('DocumentActionsService', () => {
         expect(documentListService.deleteNode).not.toHaveBeenCalled();
     });
 
-    it('should reload target upon node deletion', () => {
```

Comment:
why this test has been deleted?

Reply:
We removed 'cause now the action won't reload the target automatically

Reply:
why?